libs/pages/elewa/about-us/src/lib/about-us-location-section/about-us-location-section.component.html

```diff
@@ -0,0 +1,19 @@
+<section class="home-about-us-location-section">
+    <iframe id="map" frameborder="0" style="border:0"
+      src="https://www.google.com/maps/embed/v1/place?key=AIzaSyCRWhaEwl7eyWlByjEdvDuPMn5CCIY2C9c
+      &q=3rd Floor, The Promenade, General Mathenge Rd, Nairobi" allowfullscreen>
+    </iframe>
+    <div class="info-container">
+      <h2>How to get there</h2>
+      <p>If you want to get there, please click that button you will see a map of the place and then you can come</p>
+      <button class="directions-button"><a href="https://www.google.com/maps/place/Elewa+Company+Ltd./@-1.2539757,36.7996946,15z/data=!4m5!3m4!1s0x0:0x8a3f7edb44f2a175!8m2!3d-1.2539757!4d36.7996946" target="_blank">Get Directions
+        <div class="arrow-container"> 
+        <i class="fa fa-arrow-right"></i>
+        </div>
+      </a></button>
```

Comment:
Use the reusable animation button already created

Reply:
fixed